rotkehlchen/exchanges/binance.py

```diff
@@ -199,6 +214,7 @@ def __init__(
         self.backoff_limit = backoff_limit
         self.nonce_lock = Semaphore()
         self.offset_ms = 0
+        self.pretty_name = exchange_pretty_name
```

Comment:
Why do this? Just use self.name, which is already instantiated at the superclass `Exchange()` constructor

Reply:
I did this thinking about the front end messages displaying to the users "Binance" and "Binance US" instead of "binance" and "binance_us". I thought it would be nicer for them.

Reply:
I think I didn't see OK to overwrite the `self.name`, I will take another look at it!

Reply:
You are already doing that: https://github.com/rotki/rotki/pull/1933/files#diff-a2460a3afc4ef931badf61357148e3d25a59d58f116bc302ead8910646396d8cR206

You are passing the respective name to the constructor of super(). So no need to change anything. Just use `self.name` instead of the pretty thing.

> "Binance" and "Binance US" instead of "binance" and "binance_us". I thought it would be nicer for them.

I understand, but really not needed or worth the extra variables/code. No need to complicate things.

Reply:
well... tbh, the pretty name could be done in the front-end :D. Ok, I will roll them back.

Reply:
yes it could but also no need. Really absolutely no need to even discuss or spend time in having a different variable or logic for how the name of any exchange would appear to the user in an error message or in a log message. If it's Binance instead of binance or whatever.

For other places the frontend takes care of it already